lambdas/build-proven-user-identity-details/src/main/java/uk/gov/di/ipv/core/buildprovenuseridentitydetails/BuildProvenUserIdentityDetailsHandler.java

```diff
@@ -179,19 +175,15 @@ private NameAndDateOfBirth getProvenIdentityNameAndDateOfBirth(
 
     @Tracing
     private List<Address> getProvenIdentityAddresses(
-            List<VcStoreItem> credentialIssuerItems, List<VcStatusDto> currentVcStatuses)
+            List<VerifiableCredential> vcs, List<VcStatusDto> currentVcStatuses)
             throws ParseException, JsonProcessingException, ProvenUserIdentityDetailsException,
                     NoVcStatusForIssuerException {
-        for (VcStoreItem item : credentialIssuerItems) {
-            if (item.getCredentialIssuer().equals(ADDRESS_CRI)
+        for (var vc : vcs) {
+            if (vc.getCriId().equals(ADDRESS_CRI)
```

Comment:
Ah, I see that it the criId attribute is null then this would fail. It's the same as the logic before but might want a change if we prefer dealing in a more controlled manner